### DISCUSSION TITLE
Setup a custom subdomain in budibase app

### DISCUSSION THREAD

**Discussion by KiriakiK** [Budibase/budibase#6625] [Help]
Hi,
I have built an app on budibase and I am using docker and caddy. I have also created a subdomain under my main domain on Hostmonster and I would like to use that instead of localhost. Is something like this possible?

**Comment by melohagan** [comment]
Hi @KiriakiK 

Yes this should be possible, but you would need to host the docker container on a public server. In this case you could host it on hostmonster?

There's some docs here: https://docs.budibase.com/docs/reverse-proxy#caddy
It shows you where you can replace the domain name.
